### PR TITLE
[Site Initializer] Remove special characters in externalReferenceCode on CP 2.0

### PR DESCRIPTION
**Pull request by nNilton**

### CODE REVIEW

**Comment by nNilton** [comment]
ci:test:sf

**Comment by luanakarla** [comment]
Just starting reviewing :)

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/src/main/resources/site-initializer/client-extension-entries.json [hunk 1]
```diff
@@ -9,7 +9,7 @@
 			"[$DOCUMENT_URL:/site-initializer/documents/company/remote-app/osb-site-initializer-customer-portal/main.js$]",
 			"[$DOCUMENT_URL:/site-initializer/documents/company/remote-app/osb-site-initializer-customer-portal/runtime-main.js$]"
 		],
-		"externalReferenceCode": "ERC-001",
+		"externalReferenceCode": "ERC001",
```
Comment (joseabelenda) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/src/main/resources/site-initializer/client-extension-entries.json:12]: @nNilton let's follow Raylife's standard (RAY001) and put CP001.
ok?

**Review by joseabelenda — CHANGES_REQUESTED** [review]

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/src/main/resources/site-initializer/client-extension-entries.json (hunk 1 above)
Comment (nNilton) [reply modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/src/main/resources/site-initializer/client-extension-entries.json:12]: Ok

**Review by joseabelenda — APPROVED** [review]
Thanks @nNilton

**Comment by joseabelenda** [comment]
ci:forward

**Comment by luanakarla** [comment]
Just starting reviewing :)

**Pull request closed without merging**